.github/workflows/build_test.yml

```diff
@@ -0,0 +1,98 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  Housekeeping:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/svalinn/dagmc-ci-ubuntu-18.04-housekeeping:stable
```

Comment:
```suggestion
      image: ghcr.io/svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
```

Reply:
We want stable. They are building right now.

Reply:
Just that latest is also the default. So if you do a docker pull without anything at the end it will grab the latest one as latest is also default.